netgrasp/config/config.py

```diff
@@ -1,16 +1,15 @@
 from netgrasp.utils import debug
 
 import ConfigParser
-import sys
 
 config_instance = None
 
-class Config:
-    def __init__(self, debugger):
+class Config(object):
```

Comment:
Thanks for looking into/finding and posting a link to a good explanation.

Yes, that was a "future-proofing" change. To save potential headaches down the road.